config/initializers/delayed_job.rb

```diff
@@ -0,0 +1,9 @@
+Delayed::Worker.destroy_failed_jobs = false
```

Comment:
Should we customize some config here?

Reply:
Hey there! I think for now this looks good :)